python/paddle/fluid/layers/control_flow.py

```diff
@@ -2027,38 +2087,125 @@ def step_input(self, x, level=0):
 
     def static_input(self, x):
         """
-        Mark a variable as a RNN input. The input will not be scattered into
-        time steps. It is optional.
+        This function is used to set x as DynamicRNN's static input. It is optional.
+
+        - Case 1, set static input with LoD
+
+        .. code-block:: text
+
+            # RNN's input is the same as the case listed in step_input
+            # static input, where Si is slice data of shape [1, M]
+            x.lod = [[3, 1, 2]]
+            x.shape = [6, M]
+            x.data = [[S0],
+                      [S0],
+                      [S0],
+                      [S1],
+                      [S2],
+                      [S2]]
+
+            # step 0, batch data corresponding to the 3 input sequences
+            out.lod = [[2, 3, 1]]
+            out.shape = [6, M]
+            out.data = [[S2],
+                        [S2],
+                        [S0],
+                        [S0],
+                        [S0],
+                        [S1]]
+
+            # step 1, batch data corresponding to the 2 input sequences
+            out.lod = [[2, 3]]
+            out.shape = [5, M]
+            out.data = [[S2],
+                        [S2],
+                        [S0],
+                        [S0],
+                        [S0]]
+
+            # step 2, batch data corresponding to the 1 input sequences
+            out.lod = [[2]]
+            out.shape = [2, M]
+            out.data = [[S2],
+                        [S2]]
+
+
+        - Case 2, set static input without LoD
+
+        .. code-block:: text
+
+            # RNN's input is the same as the case listed in step_input
+            # static input, where Si is slice data of shape [1, M]
+            x.lod = [[]]
+            x.shape = [3, M]
+            x.data = [[S0],
+                      [S1],
+                      [S2]]
+
+            # step 0, batch data corresponding to the 3 input sequences
+            out.lod = [[]]
+            out.shape = [3, M]
+            out.data = [[S2],
+                        [S0],
+                        [S1]]
+
+            # step 1, batch data corresponding to the 2 input sequences
+            out.lod = [[]]
+            out.shape = [2, M]
+            out.data = [[S2],
+                        [S0]]
+
+            # step 2, batch data corresponding to the 1 input sequences
+            out.lod = [[]]
+            out.shape = [1, M]
+            out.data = [[S2]]
+
 
         Args:
-            x (Variable): The input variable.
+            x (Variable): The static input LoDTensor which should hold the same number of sequences
+                as RNN's input (the input LoDTensor set by :code:`step_input()` ). If the LoD is null,
```

Comment:
Should we save None instead of null in Python code?

Reply:
Done.